src/views/SubmissionView/hooks/useSubmissionValidationStatus.test.js

```diff
@@ -68,13 +72,13 @@ describe('useSubmissionValidationStatus', () => {
       [
         ...validationStatusArgs,
         mockSubmissionConfigValue,
-        state.setState[stateKeys.promptStatuses],
-        state.setState[stateKeys.submissionTriggered],
-        state.setState[stateKeys.fileUploadIsRequired],
+        setValue,
+        setValue,
+        setValue,
       ],
     );
     validateBeforeConfirmation.useCallback.cb();
-    expect(state.setState.submissionTriggered).toHaveBeenCalledWith(true);
+    expect(setValue).toHaveBeenCalledWith(true);
```

Comment:
It'd be great to add a comment here similar to the ones added in `FileRenderer/hooks.test.js`.